source/kraken/maintenance_worker.cpp

```diff
@@ -331,18 +325,12 @@ MaintenanceWorker::MaintenanceWorker(DataManager<type::Data>& data_manager, krak
         logger(log4cplus::Logger::getInstance(LOG4CPLUS_TEXT("background"))),
         conf(conf),
         next_try_realtime_loading(pt::microsec_clock::universal_time()){
-    try{
-        this->init_rabbitmq();
-    }catch(const std::runtime_error& ex){
-        LOG4CPLUS_ERROR(logger, "Connection to rabbitmq failed: " << ex.what());
-        data_manager.get_data()->is_connected_to_rabbitmq = false;
-    }
-    try{
-        load();
-    }catch(const std::runtime_error& ex){
-        LOG4CPLUS_ERROR(logger, "Connection to rabbitmq failed: " << ex.what());
-        data_manager.get_data()->is_connected_to_rabbitmq = false;
-    }
-}
 
+    // Load Data (.nav, disruption Bdd, build raptor data)
+    this->load_data();
+
+    // Load Realtime
+    this->load_realtime();
```

Comment:
`load_realtime` can throw exception.
And from what I can see, one will be throw everytime here.